admins/pageflow/entry.rb

```diff
@@ -153,7 +156,10 @@ def permitted_attributes
         result += Pageflow.config_for(target).admin_form_inputs.permitted_attributes_for(:entry)
 
         result += [:account_id, :theming_id] if authorized?(:read, Account)
-        result << :folder_id if authorized?(:manage, Folder)
+        if params[:id] && authorized?(:configure_folder_for, resource || resource.class)
+          result << :folder_id
+        end
```

Comment:
If `params[:id]` is present, `resource` will either return a record or cause a not found exception. The `resource.class` case should never occur.